app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -129,4 +154,13 @@ public void setCategories(List<Category> categories) {
     public void setContestDescription(String description) {
         presenter.setContestDescription(description);
     }
+
+    @Override
+    public void setNextEnabled(boolean enabled) {
+        if (enabled) {
```

Comment:
because naming is hard here.. The passive presenter callback methods sound awkward as onNext(boolean enabled); vs onNextEnabled() and onNextDisabled().